facebook/delphiFacebook/R/binary.R

```diff
@@ -378,7 +405,13 @@ get_binary_indicators <- function() {
     "smoothed_want_info_none", "weight_unif", "i_want_info_none", 6, compute_binary_response, jeffreys_binary,
     "smoothed_wwant_info_none", "weight", "i_want_info_none", 6, compute_binary_response, jeffreys_binary,
    
-    
+    # misc children
+    "smoothed_whas_child_under_18", "weight", "ch_has_child_under_18", 6, compute_binary_response, jeffreys_binary,
+
+    "smoothed_woldest_child_under_5", "weight", "ch_oldest_child_under_5", 6, compute_binary_response, jeffreys_multinomial_factory(4),
+    "smoothed_woldest_child_5_to_11", "weight", "ch_oldest_child_5_to_11", 6, compute_binary_response, jeffreys_multinomial_factory(4),
+    "smoothed_woldest_child_12_to_15", "weight", "ch_oldest_child_12_to_15", 6, compute_binary_response, jeffreys_multinomial_factory(4),
+    "smoothed_woldest_child_16_to_17", "weight", "ch_oldest_child_16_to_17", 6, compute_binary_response, jeffreys_multinomial_factory(4)
```

Comment:
Not sure why we need daily updates on these in the API, unless we want to track the COVID baby boom... seems like something for demographic contingency tables. Was this part of a list from Facebook?

Reply:
Yup, they're all from FB. I'll loop you in on Asana.

Reply:
Removed from API and weekly/monthly contingency tables with plan to include in demographic contingency tables.